clang/test/SemaSYCL/implicit-float-size-conversion.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64 -internal-isystem %S/Inputs -fsyntax-only -sycl-std=2020 -Wimplicit-float-size-conversion -verify=size-only,always-size %s
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64 -internal-isystem %S/Inputs -fsyntax-only -sycl-std=2020 -Wimplicit-float-conversion -verify=always-size,precision-only %s
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64 -internal-isystem %S/Inputs -fsyntax-only -sycl-std=2020 -Wimplicit-float-conversion -Wno-implicit-float-size-conversion -verify=prefer-precision %s
+// RUN: %clang_cc1 -fsycl-is-device -triple spir64 -internal-isystem %S/Inputs -fsyntax-only -sycl-std=2020 -Wno-implicit-float-conversion -verify %s
+
+// This test checks that floating point conversion warnings are emitted correctly when used in conjunction.
+
+#include "sycl.hpp"
+class kernelA;
+
+using namespace cl::sycl;
```

Comment:
This caused a failure in testing. Should be just `sycl`.

Reply:
https://github.com/intel/llvm/pull/6590

Reply:
@aelovikov-intel Thank you for fixing the test. I didn't know about using just the 'syc' namespace.